libres/old_tests/enkf/test_enkf_forward_init_FIELD.cpp

```diff
@@ -1,199 +0,0 @@
-/*
-   Copyright (C) 2013  Equinor ASA, Norway.
-
-   The file 'enkf_forward_init_FIELD.c' is part of ERT - Ensemble based Reservoir Tool.
-
-   ERT is free software: you can redistribute it and/or modify
-   it under the terms of the GNU General Public License as published by
-   the Free Software Foundation, either version 3 of the License, or
-   (at your option) any later version.
-
-   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
-   WARRANTY; without even the implied warranty of MERCHANTABILITY or
-   FITNESS FOR A PARTICULAR PURPOSE.
-
-   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
-   for more details.
-*/
-#include <filesystem>
-
-#include <stdlib.h>
-
-#include <ert/res_util/subst_list.hpp>
-#include <ert/util/rng.h>
-#include <ert/util/test_util.h>
-#include <ert/util/test_work_area.hpp>
-#include <ert/util/util.h>
-
-#include <ert/enkf/enkf_main.hpp>
-
-namespace fs = std::filesystem;
-
-void create_runpath(enkf_main_type *enkf_main, int iter) {
```

Comment:
I see that these c tests in general are replaced with python tests because `enkf_main_create_run_path` and `enkf_main_initialize_from_scratch` no longer exists. The tests are a bit messy in terms of purpose and are in the "old" style so I think that is ok.

However, this is leaving `libres/lib/enkf/gen_data.cpp`, `libres/lib/enkf/enkf_state.cpp` and `libres/lib/enkf/enkf_plot_gen_kw.cpp` without tests in c. Perhaps some very basic isolated unit-tests in the new c style could be added to make up for the lost c tests? When it comes to memory-debugging etc. its often a lot easier to work with c tests for c code.

Reply:
There are some tests for them (in the old style, for example: test_enkf_gen_data_config.cpp, test_enkf_gen_data_config_parse.cpp), adding some new ones is possible but not sure that the effort is worth it.

Reply:
I think you are right. Lets not dwell too much on tests for these cases.